client/overview/modal/progressive-onboarding-eligibility/index.tsx

```diff
@@ -59,75 +60,74 @@ const ProgressiveOnboardingEligibilityModal: React.FC = () => {
+							'woocommerce-payments'
+						),
+						'WooPayments'
+					),
+					components: {
+						restrictionsLink: (
+							// eslint-disable-next-line jsx-a11y/anchor-has-content
+							<a
+								rel="external noopener noreferrer"
+								target="_blank"
+								href="https://woocommerce.com/document/woopayments/startup-guide/gradual-signup/"
+							/>
+						),
+					},
+				} ) }
 			</h2>
 			<div className="wcpay-progressive-onboarding-eligibility-modal__benefits">
 				<div>
-					<Icon icon={ store } size={ 32 } />
-					<h3 className="wcpay-progressive-onboarding-eligibility-modal__benefits__subtitle">
+					<Icon icon={ store } width={ 24 } height={ 24 } />
+					<div>
+						<h3 className="wcpay-progressive-onboarding-eligibility-modal__benefits__subtitle">
+							{ __(
+								'Start selling instantly',
+								'woocommerce-payments'
+							) }
+						</h3>
 						{ __(
-							'Start selling instantly',
+							'You have a $5,000 balance limit or 30 days from your first transaction to verify and set up payouts in your account.',
 							'woocommerce-payments'
 						) }
-					</h3>
-					{ sprintf(
-						/* translators: %s: WooPayments */
-						__(
-							'%s enables you to start processing credit card payments right away.',
-							'woocommerce-payments'
-						),
-						'WooPayments'
-					) }
-				</div>
-				<div>
-					<Icon icon={ widget } size={ 32 } />
-					<h3 className="wcpay-progressive-onboarding-eligibility-modal__benefits__subtitle">
-						{ __( 'Quick and easy setup', 'woocommerce-payments' ) }
-					</h3>
-					{ __(
-						'The setup process is super simple and ensures your store is ready to accept card payments.',
-						'woocommerce-payments'
-					) }
+					</div>
 				</div>
 				<div>
-					<Icon icon={ tool } size={ 32 } />
-					<h3 className="wcpay-progressive-onboarding-eligibility-modal__benefits__subtitle">
-						{ __( 'Flexible process', 'woocommerce-payments' ) }
-					</h3>
-					{ __(
-						'You have a $5,000 balance limit or 30 days from your first transaction to verify and set up payouts in your account.',
-						'woocommerce-payments'
-					) }
+					<Icon icon={ currencyDollar } width={ 24 } height={ 24 } />
+					<div>
+						<h3 className="wcpay-progressive-onboarding-eligibility-modal__benefits__subtitle">
+							{ __(
+								'Start receiving payouts',
+								'woocommerce-payments'
+							) }
+						</h3>
+						{ __(
+							'Provide some additional details about your business so you can continue accepting payments and begin receiving payouts without restrictions.',
+							'woocommerce-payments'
+						) }
+					</div>
 				</div>
 			</div>
 			<div className="wcpay-progressive-onboarding-eligibility-modal__footer">
 				<Button variant="secondary" onClick={ handleSetup }>
-					{ __( 'Start receiving payouts', 'woocommerce-payments' ) }
+					{ __( 'Set up deposits', 'woocommerce-payments' ) }
```

Comment:
WooPayments now uses `payouts` everywhere where `deposits` would normally be used. Maybe the designs were done before this change was made, but we should keep this as `payouts` for consistency. (So, `Set up payouts`).

Reply:
Oh yes, 100%!